pkgs/applications/audio/gst123/default.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkg-config
+, wrapGAppsHook
+, gst_all_1
+, gtk3
+, ncurses
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gst123";
+  version = "0.4.1";
+
+  src = fetchFromGitHub {
+    owner = "swesterfeld";
+    repo = "gst123";
+    rev = version;
+    hash = "sha256-7qS7JJ7EY1uFGX3FxBxgH6LzK4XUoTPHR0QVwUWRz+g=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    pkg-config
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    gtk3
+    ncurses
+  ] ++ (with gst_all_1; [
+    gstreamer
+    gst-plugins-base
+    gst-plugins-good
+    gst-plugins-bad
+    gst-plugins-ugly
+    gst-libav
+  ]);
+
+  enableParallelBuilding = true;
+
+  meta = with lib; {
+    description = "GStreamer based command line media player";
+    homepage = "https://space.twc.de/~stefan/gst123.php";
+    license = licenses.lgpl2Plus;
+    maintainers = with maintainers; [ swesterfeld ];
+    inherit (ncurses.meta) platforms;
+  };
```

I have noticed in our CI it does not build on Apple/Darwin.
https://github.com/NixOS/nixpkgs/pull/248105/checks?check_run_id=15843117120

So, let's set it as broken on Darwin:

```suggestion
    inherit (ncurses.meta) platforms;
    broken = stdenv.isDarwin;
  };
```